sentry/src/main/java/io/sentry/IHub.java

```diff
@@ -96,9 +146,35 @@ public interface IHub {
    * @return The Id (SentryId object) of the event
    */
   default @NotNull SentryId captureException(@NotNull Throwable throwable) {
-    return captureException(throwable, null);
+    return captureException(throwable, new Hint());
   }
 
+  /**
+   * Captures the exception.
+   *
+   * @param throwable The exception.
+   * @param callback The callback to configure the scope for a single invocation.
+   * @return The Id (SentryId object) of the event
+   */
+  default @NotNull SentryId captureException(
+      @NotNull Throwable throwable, final @NotNull ScopeCallback callback) {
+    return captureException(throwable, null, callback);
```

Comment:
I assume this is another `null` `Hint`, if so we could replace it.
```suggestion
    return captureException(throwable, new Hint(), callback);
```